docs/manual/mkdocs.yml

```diff
@@ -202,7 +202,8 @@ nav:
         - o365group set: 'cmd/graph/o365group/o365group-set.md'
       - siteclassification:
         - siteclassification enable: 'cmd/graph/siteclassification/siteclassification-enable.md'
-        - siteclassification get: 'cmd/graph/siteclassification/siteclassification-get.md'
+        - siteclassification get: 'cmd/graph/siteclassification/siteclassification-get.md',
```

Comment:
No commas allowed at the end. It breaks the docs

Reply:
Thanks, that's really good to know 😊